### PR TITLE
ENH: make BcolzMinuteBarWriter.write take iterable

### PR DESCRIPTION
**Pull request by llllllllll**
Updates the BcolzMinuteBarWriter.write api to allow users to pass their
data as a stream instead of requiring that they loop over their data
externally. This matches the API presented by BcolzDailyBarWriter.

### CODE REVIEW

File: zipline/testing/core.py [hunk 1]
```diff
@@ -492,20 +490,17 @@ def create_data_portal(env, tempdir, sim_params, sids, adjustment_reader=None):
         )
 
 
-def write_bcolz_minute_data(env, days, path, df_dict):
+def write_bcolz_minute_data(env, days, path, data):
```
Comment (ehebert) [inline zipline/testing/core.py]: Confirming my understanding, main intent of this patch is to enable `data` here as any given iterable of `sid, frame`.

Looks good to me.

Comment (llllllllll) [inline zipline/testing/core.py]: correct. I am working on a feature where I am passing all of our writers to a user defined function and it makes it easier to understand the API if we minimize the differences between the minute and daily writers

**Comment by ssanderson** [comment]
@ehebert @llllllllll is this still something we're planning on merging?

**Comment by llllllllll** [comment]
oh, I thought this was merged, I would still like to merge this. I can rebase and merge this later assuming this still looks good.

**Comment by llllllllll** [comment]
This code is on master, it was merged in the wiki loader pr.

**Pull request closed without merging**